Update enable-previews-of-on-premises-search-results-in-cloud-hybrid-… (#983)
MicrosoftDocs/OfficeDocs-SharePoint: SharePoint/SharePointServer/hybrid/enable-previews-of-on-premises-search-results-in-cloud-hybrid-search.md

    
2. [Configure Office Online Server for SharePoint Server](/officeonlineserver/configure-office-online-server-for-sharepoint-server-2016/configure-office-online-server-for-sharepoint-server-2016)
    

> 3. Optional: Make the Office Online Server accessible outside the corporate network. This can be accomplished by configuring your corporate firewall to allow access from the public Internet to your Office Online Server. You must configure public DNS to match the `ExternalUrl` value in your Office Online Server farm. This value can be set when creating a farm using `New-OfficeWebAppsFarm -ExternalUrl <value>` or `Set-OfficeWebAppsFarm -ExternalUrl <value>`. You must use SSL when allowing access to Office Online Server from the Internet.

Reviewer: ```suggestion
3. Optional: Make the Office Online Server accessible outside the corporate network. To configure your corporate firewall to allow access from the public Internet to your Office Online Server, set the public DNS to match the `ExternalUrl` value in your Office Online Server farm. This value can be set when creating a farm using `New-OfficeWebAppsFarm -ExternalUrl <value>` or `Set-OfficeWebAppsFarm -ExternalUrl <value>`. You must use SSL when allowing access to Office Online Server from the Internet.
```

> 3. Optional: Make the Office Online Server accessible outside the corporate network. This can be accomplished by configuring your corporate firewall to allow access from the public Internet to your Office Online Server. You must configure public DNS to match the `ExternalUrl` value in your Office Online Server farm. This value can be set when creating a farm using `New-OfficeWebAppsFarm -ExternalUrl <value>` or `Set-OfficeWebAppsFarm -ExternalUrl <value>`. You must use SSL when allowing access to Office Online Server from the Internet.

Reviewer: ```suggestion
3. Optional: Make the Office Online Server accessible outside the corporate network. You can configure your corporate firewall to allow access from the public Internet to your Office Online Server. To do this, set the public DNS to match the `ExternalUrl` value in your Office Online Server farm. This value can be set when creating a farm. using `New-OfficeWebAppsFarm -ExternalUrl <value>` or `Set-OfficeWebAppsFarm -ExternalUrl <value>`. You must use SSL when you give the Internet access to your Office Online Server.
```

Author: You have a period after 'creating a farm' when there shouldn't be one based on the edit. The last sentence should not use the word 'give'. We're not giving the OOS farm anything :-)

> 3. Optional: Make the Office Online Server accessible outside the corporate network. This can be accomplished by configuring your corporate firewall to allow access from the public Internet to your Office Online Server. You must configure public DNS to match the `ExternalUrl` value in your Office Online Server farm. This value can be set when creating a farm using `New-OfficeWebAppsFarm -ExternalUrl <value>` or `Set-OfficeWebAppsFarm -ExternalUrl <value>`. You must use SSL when allowing access to Office Online Server from the Internet.

Reviewer 2: ```suggestion
3. Optional: Make the Office Online Server accessible outside the corporate network. You can configure your corporate firewall to allow access from the public Internet to your Office Online Server. To do this, set the public DNS to match the `ExternalUrl` value in your Office Online Server farm. This value can be set when creating a farm using `New-OfficeWebAppsFarm -ExternalUrl <value>` or `Set-OfficeWebAppsFarm -ExternalUrl <value>`. You must use SSL when allowing Internet access to your Office Online Server.
```
